docs/pages/machine-id/access-guides/ansible.mdx

```diff
@@ -31,7 +31,75 @@ You will need the following tools to use Teleport with Ansible.
 
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/2. Configure Ansible
+## Step 1/4. Configure RBAC
+
+First, Teleport must be configured to allow the credentials produced by the bot
+to connect to the SSH servers. In this example, access will be granted to all
+SSH hosts for the username `root`.
+
+For production use, you should use labels to restrict this access to only the
+hosts that Ansible will need to access. This is known as the principal of
+least privilege and reduces damage that exfiltrated credentials can do.
```

Comment:
I would move this paragraph after the YAML snippet so it's clearer how the user would change the label configuration.

Reply:
Addressed in 4ec7753c9f